examples/resources/hpegl_vmaas_load_balancer/nsx_t_lb.tf

```diff
@@ -1,15 +1,17 @@
 # (C) Copyright 2022 Hewlett Packard Enterprise Development LP
 
-resource "hpegl_vmaas_load_balancer" "tf_load_balancer" {
-  name  = "lab-1"       
-  description  = "creating load balancer for test"
-  enabled      =    true        
-  visibility   = "public"
-  resource_permission {
+resource "hpegl_vmaas_load_balancer" "tf_lb" {
+  name  = "lab-2"       
+  description  = "CREATE load balancer for test"
```

Comment:
please use correct names here(name, description), later these examples would be used to write documentation for this resource.

Reply:
done